controllers/manilashare_controller.go

```diff
@@ -569,16 +570,15 @@ func (r *ManilaShareReconciler) reconcileNormal(ctx context.Context, instance *m
 		return ctrl.Result{}, err
 	}
 
-	if instance.Status.ReadyCount > 0 {
+	if instance.Status.ReadyCount > 0 &&
```

Comment:
@gibizer the `ReadyCount` should be compared w/ the `instance.Spec.Replicas`, but this change will be part of a follow up.

Reply:
OK to handle this as a follow up